Automattic/woocommerce-payments
# Add way to prepopulate MCC on MOX

Fixes #8446

#### Changes proposed in this Pull Request





* Use the selected Woo Core vertical to prepopulate the selection on the MOX form (based on mapping from the platform - see https://github.com/Automattic/woocommerce-payments-server/issues/4854)





#### Testing instructions





Test 1

* Spin up a new JN site from this branch.
* During the WooCommerce Core onboarding, select a vertical (e.g. `Clothing and Accessories`)
* Start the WooPayments onboarding.
* When you are prompted to select the vertical/MCC, it should be prepoulated based on the value you selected in the Core onboarding, using the following mapping:

```
[
	'clothing_and_accessories'  => 'retail__clothing_and_apparel',
	'health_and_beauty'         => 'retail__beauty_products',
	'food_and_drink'            => 'food_and_drink__other_food_and_dining',
	'home_furniture_and_garden' => 'retail__home_furnishings_and_furniture',
	'education_and_learning'    => 'education__other_educational_services',
	'electronics_and_computers' => 'digital_products__other_digital_goods',
];
```

Test 2 

* Spin up another new JN site.
* This time, skip the WooCommerce Core onboarding, not choosing an industry.
* The WooPayments MOX should _not_ prepoulate the value, instead it should be the default `Select an option`
-------------------


**Post merge**
